interfaces/openapi-to-go-server/example/api/rid/interface.gen.go

```diff
@@ -7,54 +7,57 @@ import (
 )
 
 var (
-	SearchIdentificationServiceAreasSecurity = map[string]api.SecurityScheme{
-		"AuthFromAuthorizationAuthority": []api.AuthorizationOption{
-			{RequiredScopes: []string{"dss.read.identification_service_areas"}},
+	SearchIdentificationServiceAreasSecurity = []api.AuthorizationOption{
+		{
+			"AuthFromAuthorizationAuthority": {"dss.read.identification_service_areas"},
 		},
 	}
-	GetIdentificationServiceAreaSecurity = map[string]api.SecurityScheme{
-		"AuthFromAuthorizationAuthority": []api.AuthorizationOption{
-			{RequiredScopes: []string{"dss.read.position_reporting_entities"}},
+	GetIdentificationServiceAreaSecurity = []api.AuthorizationOption{
+		{
+			"AuthFromAuthorizationAuthority": {"dss.read.identification_service_areas"},
 		},
 	}
-	CreateIdentificationServiceAreaSecurity = map[string]api.SecurityScheme{
-		"AuthFromAuthorizationAuthority": []api.AuthorizationOption{
-			{RequiredScopes: []string{"dss.write.identification_service_areas"}},
+	CreateIdentificationServiceAreaSecurity = []api.AuthorizationOption{
+		{
+			"AuthFromAuthorizationAuthority": {"dss.write.identification_service_areas"},
 		},
 	}
-	UpdateIdentificationServiceAreaSecurity = map[string]api.SecurityScheme{
-		"AuthFromAuthorizationAuthority": []api.AuthorizationOption{
-			{RequiredScopes: []string{"dss.write.identification_service_areas"}},
+	UpdateIdentificationServiceAreaSecurity = []api.AuthorizationOption{
+		{
+			"AuthFromAuthorizationAuthority": {"dss.write.identification_service_areas"},
 		},
 	}
-	DeleteIdentificationServiceAreaSecurity = map[string]api.SecurityScheme{
-		"AuthFromAuthorizationAuthority": []api.AuthorizationOption{
-			{RequiredScopes: []string{"dss.write.identification_service_areas"}},
+	DeleteIdentificationServiceAreaSecurity = []api.AuthorizationOption{
+		{
+			"AuthFromAuthorizationAuthority": {"dss.write.identification_service_areas"},
 		},
 	}
-	SearchSubscriptionsSecurity = map[string]api.SecurityScheme{
-		"AuthFromAuthorizationAuthority": []api.AuthorizationOption{
-			{RequiredScopes: []string{"dss.read.identification_service_areas"}},
+	SearchSubscriptionsSecurity = []api.AuthorizationOption{
+		{
+			"AuthFromAuthorizationAuthority": {"dss.read.identification_service_areas"},
 		},
 	}
-	GetSubscriptionSecurity = map[string]api.SecurityScheme{
-		"AuthFromAuthorizationAuthority": []api.AuthorizationOption{
-			{RequiredScopes: []string{"dss.read.position_reporting_entities", "dss.write.position_reporting_entities"}},
+	GetSubscriptionSecurity = []api.AuthorizationOption{
```

Comment:
This output is what we want, but when I run `make apis` on this branch, I get the two-scope rendering here -- I think that's because the interfaces/rid/v1 subrepo reference hasn't been moved to [975ec682a4c976abfe8ee29772768940d79044e0](https://github.com/uastech/standards/tree/fix/auth-scopes).  Could we either update that subrepo reference (if doing so doesn't ripple out to other parts of the repo), or else update this file to reflect the output of `make apis` at the current subrepo reference?

Reply:
> or else update this file to reflect the output of make apis at the current subrepo reference

I will go down this road to keep the consistency of the committed code.

Reply:
Done. I will include the update to the generated code in the other PR :)